images/scripts/services.setup

```diff
@@ -43,7 +44,7 @@ docker run -d --rm --name freeipa -ti -h f0.cockpit.lan \
     -p 53:53/udp -p 53:53 -p 80:80 -p 443:443 -p 389:389 -p 636:636 -p 88:88 -p 464:464 -p 88:88/udp -p 464:464/udp -p 123:123/udp \
     -v /var/lib/ipa-data:/data:Z \
     -v /sys/fs/cgroup:/sys/fs/cgroup:ro \
-    freeipa/freeipa-server:fedora-rawhide \
+    quay.io/freeipa/freeipa-server:fedora-rawhide \
```

Comment:
The comment above "docker pull quay.io/freeipa/freeipa-server:fedora-rawhide" should probably also be updated to mention quay.io.

Reply:
Done.